src/pages/home/report/ReportActionsView.js

```diff
@@ -63,18 +63,17 @@ class ReportActionsView extends React.Component {
 
         this.didLayout = false;
         this.didSubscribeToReportTypingEvents = false;
-
         this.unsubscribeVisibilityListener = null;
 
+        // We need this.sortedAndFilteredReportActions to be set before this.state is initialized because the function to calculate the newMarkerReportActionID uses the sorted report actions
+        this.sortedAndFilteredReportActions = this.getSortedReportActionsForDisplay(props.reportActions);
```

Comment:
Commented above, but this component is also responsible for loading the report actions - so how do we know that `props.reportActions` are up to date at this point?

Reply:
I suppose we don't. It will load the marker based on the data it has and then the callback gets called when new report actions come into onyx. But I think further improvements can be handled via this convo: https://expensify.slack.com/archives/C01GTK53T8Q/p1671704053216749

Reply:
Yeah, I was doing some testing on this last week and think the design is broken in the same way on staging/production today anyway.

Reply:
Ok cool. Gonna resolve then.